### PR TITLE
[ansible] Distributed volumes

### PR DESCRIPTION
**Pull request by danielapetito**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
NFS volume

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:
dojot/dojot#2338

### CODE REVIEW

File: inventories/example_local/hosts.yaml [hunk 1]
```diff
@@ -41,12 +43,15 @@ all:
         dojot_nodes:
           hosts:
             worker_host: #host for dojot related services volumes (apigw; influxdb; kafka_ws; minio; mongodb; postgres; prometheus;)
+            # nfs_host: #host for dojot related services volumes. If you are using NFS volume with NFS SERVER you should uncomment
```
[the remark this answers is not in the document]
Comment (gabrielifg) [reply inventories/example_local/hosts.yaml:46]: eles não podem ser usados em conjunto ? @marianoleonardo

[the remark this answers is not in the document]
Comment (danielapetito) [reply inventories/example_local/hosts.yaml:46]: não.. deve-se escolher uma opção local-storage ou nfs

**Review by ewerton-silva00 — APPROVED** [review]
Working Ok. Great job!

File: inventories/example_local/hosts.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielapetito) [reply inventories/example_local/hosts.yaml]: atualizado mensagem conforme sugestão @marianoleonardo

**Review by ewerton-silva00 — APPROVED** [review]

**Review by gabrielifg — APPROVED** [review]
Niceee !!!!

**Review by marianoleonardo — APPROVED** [review]
Great Job!!!👏🏼👏🏼👏🏼

**Pull request merged**